src/sonic-config-engine/sonic-cfggen

```diff
@@ -42,7 +42,7 @@ from portconfig import get_port_config
 from sonic_device_util import get_machine_info
 from sonic_device_util import get_platform_info
 from sonic_device_util import get_system_mac
-from sonic_device_util import get_npu_id_from_name
+from sonic_device_util import get_npu_id_from_name, get_npu_pci_device_id
```

Comment:
Can you move the new import  to new line

Reply:
shall update.

Reply:
Done